sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -493,27 +494,7 @@ private synchronized void triggerIngestion(final @NonNull GroupState groupState)
 
         /* Remember this batch. */
         groupState.mSendingBatches.put(batchId, batch);
-
-        /*
-         * Due to bug on old Android versions (verified on 4.0.4),
```

Comment:
Was the async bug tested on 4.1? The link to the android source code is not clear about released versions, just wanting to make sure.

Reply:
Yes (on emulator). Also, that commit (android source) contains comment about the versions:

> The AsyncTask class must be loaded on the UI thread. This is done
> automatically as of {@link android.os.Build.VERSION_CODES#JELLY_BEAN}.